doc/guide/start.rst

```diff
@@ -0,0 +1,453 @@
+
+.. code-block:: console
+
+  $ srun -N2 flux start ./workload.sh
+  srun: job 1505795 queued and waiting for resources
+  srun: job 1505795 has been allocated resources
+   10:07:12 run 4.6s,  owner alice,  depth 0,  size 2
+  2 Nodes, 72 Cores, 0 GPUs
+  0: quartz25
+  1: quartz26
+
+Finally, Slurm batch execution of a Flux workload can be accomplished
+by wrapping the srun command in a batch script we will name ``batch.sh``:
+
+.. code-block:: shell
+
+  #!/bin/sh
+  #SBATCH -N2
+  srun flux start ./workload.sh
+
+Whee!
+
+.. code-block:: console
+
+  $ sbatch ./batch.sh
+  Submitted batch job 1505846
+  $ cat slurm-1505846.out
+   10:09:57 run 4.8s,  owner alice,  depth 0,  size 2
+  2 Nodes, 72 Cores, 0 GPUs
+  0: quartz47
+  1: quartz48
+  $
+
+Inserting Flux between Slurm and a real workload script or workflow executor
+could have some advantages, such as:
+
+- Workload scripts that use Flux commands or the Flux python API can be made
+  portable to other systems that don't use Slurm as their primary resource
+  manager, as long as they can launch a Flux instance.
+
+- High throughput workloads don't have to bother the central Slurm controller
+  with every step, so they are less of a burden on the system and may run
+  faster.
+
+- When combined with the `flux-sched project
+  <https://github.com/flux-framework/flux-sched>`_, Flux offers more
+  sophisticated step scheduling.
+
+- A Flux instance is highly configurable to meet the specific demands of
+  the workload.
+
+To be clear, Flux does not have "steps" per se, only jobs.   We don't need them
+since Flux batch jobs are just regular Flux jobs that happen to be independent
+Flux instances, and Flux instances have the capability to run more Flux jobs.
+
+Since Slurm runs on many of the largest HPC systems, the capability to launch
+Flux instances under Slurm provided early opportunities to work on Flux
+portability and scalability on those systems without requiring much buy-in from
+the system owners, other than permission to run there.
+
+Starting with Flux
+------------------
+
+Flux can run parallel programs, and a Flux instance can be run as one, so Flux
+can run Flux.  No surprise there.  What does that look like and how is it
+leveraged to provide expected workload manager abstractions?
+
+First let's try our workload commands in the "outer" Flux instance, in this
+case a standalone system instance, although this section applies to all Flux
+instances.
+
+.. code-block:: console
+
+  $ flux uptime
+   10:13:27 run 23d,  owner flux,  depth 0,  size 101,  7 drained,  8 offline
+  $ flux resource info
+  98 Nodes, 392 Cores, 0 GPUs
+  $ flux run -N2 hostname
+  $ flux run --label-io -N2 hostname
+  0: fluke6
+  1: fluke7
+
+The instance depth is zero since this is the system instance.  We also get a
+heads up about some unavailable nodes, and note that the instance owner is
+the ``flux`` user, but otherwise this looks like before.  Now, an interactive
+Flux allocation:
```

Comment:
here and maybe this section, can show depth via `flux getattr instance-level`?

Reply:
`flux uptime` is already showing the depth in each example.  I wanted to stay focused on starting flux and not get that too diluted introducing other stuff.